build/configs/artik053/STDK/defconfig

```diff
@@ -916,7 +916,6 @@ CONFIG_SMARTFS_ERASEDSTATE=0xff
 CONFIG_SMARTFS_MAXNAMLEN=32
 # CONFIG_SMARTFS_MULTI_ROOT_DIRS is not set
 CONFIG_SMARTFS_ALIGNED_ACCESS=y
-# CONFIG_SMARTFS_BAD_SECTOR is not set
```

Comment:
@Taejun-Kwon,  i understand BAD sector logic is not used, but should we remove this code completely?

Reply:
we will not use it stand alone anymore, will be added inside of crc feature later.